InWorldz/InWorldz.PhysxPhysics/PhysxScene.cs

```diff
@@ -278,7 +278,7 @@ public override void Initialize(IMesher meshmerizer, Nini.Config.IConfigSource c
 
             Nini.Config.IConfig startupcfg = config.Configs["Startup"];
             if (startupcfg != null)
-                _gridmode = startupcfg.GetBoolean("gridmode", true);
+                _gridmode = startupcfg.GetBoolean("gridmode", false);
```

Comment:
This brings PhysX's default into line with everywhere else in the code that reads the gridmode.